src/services/syncFeedPerUser.js

```diff
@@ -208,19 +208,19 @@ const syncFeedPerUserProcess = async (userId, source = '') => {
   // Get f2 user to follow
   await syncMainFeedF2(userId);
   // Get broad user to follow
-  await syncMainFeedBroad(userId, source);
+  // await syncMainFeedBroad(userId, source);
 };
 
-const syncMainFeedBroadPerUser = async (req, res) => {
-  try {
-    const {userId} = req.body;
-    await syncMainFeedBroad(userId);
+// const syncMainFeedBroadPerUser = async (req, res) => {
+//   try {
+//     const {userId} = req.body;
+//     await syncMainFeedBroad(userId);
 
-    return successResponse(res, 'sync data sucesfully', []);
-  } catch (error) {
-    return errorResponse(res, error.toString(), 500);
-  }
-};
+//     return successResponse(res, 'sync data sucesfully', []);
+//   } catch (error) {
+//     return errorResponse(res, error.toString(), 500);
+//   }
+// };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `syncMainFeedBroadPerUser` has been commented out. If this function is not going to be used anymore, it's better to remove it completely instead of leaving it as commented-out code.

```diff
- // const syncMainFeedBroadPerUser = async (req, res) => {
- //   try {
- //     const {userId} = req.body;
- //     await syncMainFeedBroad(userId);
-  
- //     return successResponse(res, 'sync data sucesfully', []);
- //   } catch (error) {
- //     return errorResponse(res, error.toString(), 500);
- //   }
- // };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->